android/app/src/bt/java/covidsafepaths/bt/exposurenotifications/DebugMenuModule.java

```diff
@@ -67,7 +85,7 @@ public void onSuccess(Boolean result) {
 
             @Override
             public void onFailure(Throwable t) {
-                callback.invoke(t.getMessage(), null);
+                //callback.invoke(t.getMessage(), null);
```

Comment:
Should we just delete the code instead? We could always revisit with the magic 🧙‍♀️ of git later